.editorconfig

```diff
@@ -54,3 +54,6 @@ csharp_new_line_before_catch = true
 csharp_new_line_before_finally = true
 csharp_new_line_before_members_in_object_initializers = true
 csharp_new_line_before_members_in_anonymous_types = true
+
+# CS0168: Variable is declared but never used
+dotnet_diagnostic.CS0168.severity = none
```

Comment:
Can we set this to `Info` level instead of turning it off?

Reply:
That is the default - should i suppress this for their code instead?

Reply:
we could just ignore the warnings? I don't know how bad it is... also should be an OK example for a quick bulk refactor

Reply:
was avoiding making changes to their code , but now set it to suggestion(Info) hopefully that fixes the CI build.